### DISCUSSION TITLE
Use the plugin in themes for Shopify theme store

### DISCUSSION THREAD

**Discussion by slavafyi** [barrel/shopify-vite#84] [Q&A] [answered]
Hi everyone who is here :)

I would like to use the plugin in my theme for Shopify theme store. I'm just getting into development planning, so I want to choose the right path. I have doubts that shopify team will approve the use of asset files with hash. Like `theme-[hash].js`. 

I remember seeing someone previously discussing this in issues, but I couldn't find it. Would love to hear some insights.

I'll try to contact shopify support if I don't get a response here. Because I checked the last published themes in the store and didn't find this

**Comment by montalvomiguelo** [comment, marked as answer]
I know @thomaskimura uses it in [Paper Theme](https://themes.shopify.com/themes/paper/styles/poster). He also provides the [toolkit](https://github.com/BrickspaceLab/paper-toolkit) to customize it, so cool 😎

**Reply by slavafyi** [reply]
Yeah really, it's strange that I didn't see it. Thanks!

So it shouldn't have any problem, I guess :)
